assignment-2/src/presenter/QuestionListPresenter.js

```diff
@@ -0,0 +1,58 @@
+import questionModel from "../model/QuestionModel.js"
+import model from "../model/UserModel.js";
+
+class QuestionListPresenter {
+    onSearchBarChange(tagOrTitle) {
+        questionModel.changeTitleOrTagSearch(tagOrTitle);
+    }
+
+    onClickSearchTitle() {
+        // questionModel.state.questions.filter( (question, index, arr) => (
+        //     question.title.includes(questionModel.state)
+        // ))
+        let questionList = questionModel.state.questions.filter((question, index, arr) => (
```

Comment:
You do not need to declare the `index` and `arr` parameters if you don't need them. In JS, there are no "method signatures": any function accepts any number of parameters.